data/mods/ssb/statuses.ts

```diff
@@ -38,6 +38,18 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 	IMPORTANT: Obtain the username from getName
 	*/
 	// Please keep statuses organized alphabetically based on staff member name!
+	elgino: {
+		noCopy: true,
+		onStart() {
+			this.add(`c|${getName('Elgino')}|Time to save Hyrule!`);
```

Comment:
make sure to add a message that shows the new type change.

Reply:
Done